File: sdkwork-ai-sdk-typescript/src/types/open-ai-file-list.ts
import type { OpenAiFileObject } from './open-ai-file-object';

/** OpenAI标准文件列表对象 */
export interface OpenAiFileList {
  /** 文件列表数据 */
  data?: OpenAiFileObject[];
  /** 对象类型 */
  object?: string;
  /** 第一条记录的ID */
  first_id?: string;
  /** 最后一条记录的ID */
  last_id?: string;
  /** 是否有更多数据 */
  has_more?: boolean;
}

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/open_ai_file_list.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class OpenAiFileList:
    """OpenAI标准文件列表对象"""
    data: List[OpenAiFileObject] = None
    object: str = None
    first_id: str = None
    last_id: str = None
    has_more: bool = None
